src/math/NewtonRaphson.h

```diff
@@ -27,6 +28,33 @@ struct DoubleSolver
     }
 };
 
+//! @brief Wrapper for Eigen::SparseMatrix
```

Comment:
Why do we need this "wrapping"?

Reply:
Our newton raphson algortihm requests a solver with the interface `Vector Solve(Matrix, Vector)` which is not provided by `Eigen` out of the box. Thus, it is wrapped here. But this is _very_ similar to the [`EigenSparseSolver`](https://github.com/nutofem/nuto/blob/PDE_reviewed/src/math/EigenSparseSolve.h#L38) and could probably be removed.